Exo-1-es6/gulpfile.js

```diff
@@ -0,0 +1,28 @@
+var gulp = require('gulp');
```

Comment:
Gulp should not be necessary. I'd recommand reading :
https://github.com/FrontenderMagazine/task_automation_with_npm_run/blob/master/eng.md
http://naholyr.fr/2015/11/ecrire-des-scripts-npm-multi-plateforme/
https://gist.github.com/oncletom/1e233f2100c4e0877922